plugins/modules/azure_rm_manageddisk.py

```diff
@@ -462,6 +490,10 @@ def __init__(self):
             disk_m_bps_read_write=dict(
                 type='int'
             ),
+            tier=dict(
+                type='str',
+                choices=['', 'P1', 'P2', 'P3', 'P4', 'P6', 'P10', 'P15', 'P20', 'P30', 'P40', 'P50', 'P60', 'P70', 'P80']
```

Comment:
Or change the document, thank you!
```suggestion
                choices=['P1', 'P2', 'P3', 'P4', 'P6', 'P10', 'P15', 'P20', 'P30', 'P40', 'P50', 'P60', 'P70', 'P80']
```